src/less-style/structure.less

```diff
@@ -37,6 +37,23 @@ body.vellum-full-screen {
 
                 }
 
+                .fd-commander {
+                    display: flex;
+                    padding: 6px 4px 6px 12px;
+                    background-color: @cc-dark-cool-accent-mid;
+
+                    .fd-add-question-toggle {
+                        color: white;
+                        height: 17px;
+                    }
+
+                    input {
+                        border: 1px solid @cc-dark-cool-accent-low;
+                        flex: 2;
+                        margin: -2px 0;
```

Comment:
By "above add question" I meant something like the below image. But I'm just thinking about possibilities, especially for the future. Replacing the add question with the commander and adding an X to dismiss it does sound like a reasonable compromise.

<img width="324" alt="screen shot 2017-08-22 at 5 33 45 pm" src="https://user-images.githubusercontent.com/1486591/29588584-208ab1aa-8760-11e7-9a28-aaec3d1af937.png">